src/tool/python/code_writers.h

```diff
@@ -1317,6 +1379,7 @@ static PyObject* @_get_%(%* self, void* /*unused*/)
                 field.Parent().TypeName(),
                 field.Name(),
                 bind<write_winrt_wrapper>(field.Parent()),
+                bind<write_struct_property_get_variable>(field),
                 bind<write_struct_property_get_variable>(field));

```

Comment:
Is this an intentional duplicate?

Reply:
Yes, but it seems kind of ugly looking at it again. I switched away from using if constexpr branches in favor of just having converter<T> type specializations. However, that meant I couldn't use type deduction anymore. The variable is duplicated here because I'm using declype instead of specifying the type directly.

example: `return py::converter<decltype(self->obj.Height)>::convert(self->obj.Height);`

I kept the convert_to<T> helper, I'm thinking I should have kept the convert<T> helper as well. Thoughts?